crates/diesel_models/src/query/business_profile.rs

```diff
@@ -46,7 +46,7 @@ impl BusinessProfile {
     ) -> StorageResult<Self> {
         generics::generic_find_one::<<Self as HasTable>::Table, _, _>(
             conn,
-            dsl::profile_id.eq(profile_id.to_owned()),
+            dsl_identifier.eq(profile_id.to_owned()),
```

Comment:
Can we have separate v1 and v2 functions, for more clarity as there's no profile_id in v2